### ISSUE TITLE
LoadError when requiring nats/io/client

### ISSUE THREAD

**Issue by Josh-Meyer** [nats-io/nats-pure.rb#63]
On version `2.0.0.pre.rc1`, ruby version 2.7.4 via rbenv, and bundler version 2.1.4.

Trying `require 'nats'` and `require 'nats/client'` also result in the same issue, which makes sense given that the first thing both of them do is `require 'nats/io/client'`.

The direct cause is that the `lib/nats/io/kv.rb` file is missing from my local installation. I checked, and it does exist in both the [tagged branch](https://github.com/nats-io/nats-pure.rb/blob/v2.0.0-rc1/lib/nats/io/kv.rb) and in the zipped source. So, I'm not sure what's going on. I was able to patch the issue locally by manually copy-pasting the file from github, but that's not an option in production.

```
irb(main):001:0> require 'nats/io/client'
Traceback (most recent call last):
       16: from ... /.rbenv/versions/2.7.4/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
       15: from ... /.rbenv/versions/2.7.4/lib/ruby/2.7.0/bundler/cli.rb:476:in `exec'
       14: from ... /.rbenv/versions/2.7.4/lib/ruby/2.7.0/bundler/cli/exec.rb:28:in `run'
       13: from ... ./rbenv/versions/2.7.4/lib/ruby/2.7.0/bundler/cli/exec.rb:63:in `kernel_load'
       12: from ... /.rbenv/versions/2.7.4/lib/ruby/2.7.0/bundler/cli/exec.rb:63:in `load'
       11: from ... /.rbenv/versions/2.7.4/bin/irb:23:in `<top (required)>'
       10: from ... /.rbenv/versions/2.7.4/bin/irb:23:in `load'
        9: from ... /.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/irb-1.2.6/exe/irb:11:in `<top (required)>'
        8: from (irb):1
        7: from (irb):1:in `require'
        6: from ... /.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/nats-pure-2.0.0.pre.rc1/lib/nats/io/client.rb:18:in `<top (required)>'
        5: from ... /.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/nats-pure-2.0.0.pre.rc1/lib/nats/io/client.rb:18:in `require_relative'
        4: from ... /.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/nats-pure-2.0.0.pre.rc1/lib/nats/io/msg.rb:14:in `<top (required)>'
        3: from ... /.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/nats-pure-2.0.0.pre.rc1/lib/nats/io/msg.rb:14:in `require_relative'
        2: from ... /.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/nats-pure-2.0.0.pre.rc1/lib/nats/io/js.rb:17:in `<top (required)>'
        1: from ... /.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/nats-pure-2.0.0.pre.rc1/lib/nats/io/js.rb:17:in `require_relative'
LoadError (cannot load such file -- ... /.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/nats-pure-2.0.0.pre.rc1/lib/nats/io/kv)
```

**Comment by wallyqs** [comment]
Thanks for the report, fixing this asap.

**Comment by wallyqs** [comment]
Sorry for the inconvenience, this should be fixed now on the rc2 version of the gem. 
Fixed via #64
